src/java/org/apache/cassandra/config/AccordSpec.java

```diff
@@ -194,15 +194,17 @@ public enum TransactionalRangeMigration
     public boolean ephemeralReadEnabled = true;
     public boolean state_cache_listener_jfr_enabled = true;
     public final JournalSpec journal = new JournalSpec();
-    public final MinEpochRetrySpec minEpochSyncRetry = new MinEpochRetrySpec();
-
-    public static class MinEpochRetrySpec extends RetrySpec
-    {
-        public MinEpochRetrySpec()
+    public final RetrySpec minEpochSyncRetry = new RetrySpec() {
```

Comment:
is `org.apache.cassandra.config.DatabaseDescriptorRefTest` clean?

Reply:
`"org.apache.cassandra.config.Config$1",` stuff like this is likely to break `DatabaseDescriptorRefTest`